relentless/simulate/generic.py

```diff
@@ -0,0 +1,296 @@
+import importlib
+import inspect
+
+import numpy as np
+
+from . import simulate
+
+class SimulationOperationAdapter(simulate.SimulationOperation):
```

Comment:
Sorry to request this cosmetic change, but could you rename this class to `GenericOperation`? I think that will make its intent clearer.